README.md

```diff
@@ -33,19 +33,29 @@ module "rds-snapshot-cleaner" {
 ```
 
 <!-- BEGINNING OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
```

Comment:
Just move this to the top of the file, then you don't need to delete the text in `main.tf`.